Description: Add note about installing perl-modules on Debian to
 generated manpage
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Forwarded: not-needed
Last-Update: 2013-09-28

--- a/src/eximstats.src
+++ b/src/eximstats.src
@@ -501,6 +501,10 @@
  make test
  make install
 
+On B<Debian GNU/Linux> you can use
+C<apt-get install libgd-perl libgd-text-perl libgd-graph-perl>
+instead.
+
 =item B<-chartdir>I <dir>
 
 Create the charts in the directory <dir>
